server/pulp/server/controllers/importer.py

```diff
@@ -276,6 +276,8 @@ def update_importer_config(repo_id, importer_config):
     for k, v in importer_config.iteritems():
         if v is not None:
             repo_importer.config[k] = v
+            if k == importer_constants.KEY_FEED:
+                repo_importer.config[importer_constants.KEY_FEED_UPDATED] = True
```

Comment:
@ggainey any reason we placed 'feed-updated' into the importer's config and not scratchpad?
Importer scratchpad is not editable by the user, meanwhile config yes.
```ipanova@ina pulp]$ curl -k  -H "Content-Type:application/json" -X PUT -u admin:admin -d '{"importer_config":{"feed-updated":"data"}}' https://localhost/pulp/api/v2/repositories/zoo1/importers/yum_importer/ | python -m json.tool
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   215  100   172  100    43   1202    300 --:--:-- --:--:-- --:--:--  1503
{
    "error": null,
    "result": null,
    "spawned_tasks": [
        {
            "_href": "/pulp/api/v2/tasks/45b691eb-ff57-4633-ab0c-fe57ab1937e4/",
            "task_id": "45b691eb-ff57-4633-ab0c-fe57ab1937e4"
        }
    ]
}
[ipanova@ina pulp]$ curl -k  -H "Content-Type:application/json" -X GET -u admin:admin https://localhost/pulp/api/v2/repositories/zoo1/importers/ | python -m json.tool
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   560  100   560    0     0   6021      0 --:--:-- --:--:-- --:--:--  6021
[
    {
        "_href": "/pulp/api/v2/repositories/zoo1/importers/yum_importer/",
        "_id": {
            "$oid": "5d31d48045ef48435cb5aaf0"
        },
        "_ns": "repo_importers",
        "config": {
            "feed": "https://repos.fedorapeople.org/pulp/pulp/demo_repos/zoo1/",
            "feed-updated": "data"
        },
        "id": "yum_importer",
        "importer_type_id": "yum_importer",
        "last_override_config": {},
        "last_sync": "2019-07-19T14:33:10Z",
        "last_updated": "2019-07-22T10:02:27Z",
        "repo_id": "zoo1",
        "scratchpad": {
            "repomd_checksum": "e221dd72a3b235340d544a890432fd173bc451999b708b9c3c4b13da61ee7d9c",
            "repomd_revision": 1331832478
        }
    }
]

```

Reply:
Yes - I didn't know 'scratchpad' exists :)  Will change.

Reply:
Moved into scratchpad, using ImporterScratchPadMixin